inspire_matcher/api.py

```diff
@@ -35,7 +35,12 @@
 
 
 def _get_validator(validator_param):
-    if callable(validator_param):
+    all_true = True
+    for param in validator_param:
```

Comment:
so if we don't want to convert all our configs you need to handle the case when validator is not a list and put it inside one.